File: DBPractice/app/src/main/java/com/leeddev/dbpractice/MainActivity.java
package com.leeddev.dbpractice;

import android.os.Bundle;
import android.widget.Button;
import android.widget.EditText;
import android.widget.Toast;

import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

    // creating variables for our edittext, button and dbhandler
   private Dbhelper dbhelper;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        dbhelper=new Dbhelper(this);
        dbhelper.insertContact("32.4","33","234");
        Toast.makeText(this, "Inserted", Toast.LENGTH_SHORT).show();

    }

}

File: SpeedoMeter3/app/src/main/java/com/leeddev/speedometer/MainActivity.kt
package com.leeddev.speedometer

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.ImageView

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        var ivHome:ImageView=findViewById(R.id.iv_home)
        var button:Button=findViewById(R.id.button)
        button.setOnClickListener {
            ivHome.animate().rotationX(100f).duration = 100
        }
    }
}